apps/client/src/app/modules/list/data/ListRowSerializationHelper.ts

```diff
@@ -0,0 +1,268 @@
+import { GarlandToolsService } from '../../../core/api/garland-tools.service';
+import { LocalizedDataService } from '../../../core/data/localized-data.service';
+import { I18nToolsService } from '../../../core/tools/i18n-tools.service';
+import { I18nName } from '../../../model/common/i18n-name';
+import { MobNamePipe } from '../../../pipes/pipes/mob-name.pipe';
+import { VenturesComponent } from '../../item-details/ventures/ventures.component';
+import { DataType } from '../data/data-type';
+import { getItemSource, ListRow } from '../model/list-row';
+
+export class ListRowSerializationHelper {
+
+  public static getData<T = any>(row: ListRow, type: DataType, isObject = false): T {
+    return getItemSource<T>(row, type, isObject);
+  }
+
+  constructor(
+    private i18nTools: I18nToolsService,
+    private l12n: LocalizedDataService,
+    private gt: GarlandToolsService,
+  ) {
+  }
+  public applyItemName(obj) {
```

Comment:
You need to define return type, even if it's void, to make things easier to follow and to have the compiler properly understand what you're doing.

Reply:
currently it accepts anything that has an id or idName and returns whatever the input is so should I make it semi explit or just put `any`